slack/web/classes/blocks.py

```diff
@@ -104,105 +179,171 @@ def __init__(
         *,
         image_url: str,
         alt_text: str,
-        title: Optional[str] = None,
+        title: Optional[Union[str, dict, TextObject]] = None,
         block_id: Optional[str] = None,
+        **others: dict,
     ):
+        """A simple image block, designed to make those cat photos really pop.
+        https://api.slack.com/reference/block-kit/blocks#image
         """
-        A simple image block, designed to make those cat photos really pop.
-
-        https://api.slack.com/reference/messaging/blocks#image
-
-        Args:
-            image_url: Publicly hosted URL to be displayed. Cannot exceed 3000
-                characters.
-            alt_text: Plain text summary of image. Cannot exceed 2000 characters.
-            title: A title to be displayed above the image. Cannot exceed 2000
-                characters.
-            block_id: ID to be used for this block - autogenerated if left blank.
-                Cannot exceed 255 characters.
-        """
-        super().__init__(subtype="image", block_id=block_id)
+        super().__init__(type=self.type, block_id=block_id)
+        show_unknown_key_warning(__name__, others)
+
         self.image_url = image_url
         self.alt_text = alt_text
-        self.title = title
+        self.title = TextObject.parse(title)
 
     @JsonValidator(
         f"image_url attribute cannot exceed {image_url_max_length} characters"
     )
-    def image_url_length(self):
+    def validate_image_url_length(self):
```

Comment:
I've noticed Rodney's WIP implementation had a conflict with class fields. I've added `validate_` prefix to all the methods with `@JsonValidator` decorator. The naming convention can be `_validate_` to indicate they're supposed to be used internally.

Reply:
renamed with `_validate_` prefix